src/Modules/Build.lua

```diff
@@ -1224,6 +1225,9 @@ function buildMode:CompareStatList(tooltip, statList, actor, baseOutput, compare
 			local statVal1 = compareOutput[statData.stat] or 0
 			local statVal2 = baseOutput[statData.stat] or 0
 			local diff = statVal1 - statVal2
+			if statData.stat == "FullDPS" and not GlobalCache.useFullDPS and not self.viewMode == "TREE" then
+				diff = 0
+			end
```

Comment:
When does this block apply and why do we have to manually disable the tooltip?  (I assume that's the intent, here)

Reply:
When doing stat compares the `baseOutput` is calculated at the start of every re-calculation. Then, when doing a compare, the `MiscCalculator` is used to generate a `compareOutput`. In the cases where FullDPS is not selected as the comparison field the `compareOutput.fullDPS` does not exist and hence the stat compare reports that you are losing all of your FullDPS value. This prevents that.
In the case of a single `node` or `nodePath` compare in Tree view it will always be there.